pandas/tests/tools/test_to_numeric.py

```diff
@@ -731,17 +731,23 @@ def test_to_numeric_from_nullable_string(values, expected):
     "data, input_dtype, downcast, expected_dtype",
     (
         ([1, 1], "Int64", "integer", "Int8"),
+        ([1.0, 1.0], "Float64", "integer", "Int8"),
```

Comment:
```suggestion
        ([1.0, pd.NA], "Float64", "integer", "Int8"),
```

Otherwise it is identical to the last case